lib/package.py

```diff
@@ -30,8 +30,7 @@
 CONF = config.get_config().CONF
 LOG = logging.getLogger(__name__)
 
-# TODO: make this configurable by a base dir parameter
-BUILD_CACHE_DIR = "cache"
+BUILD_CACHE_DIR = os.path.join(CONF.get('default').get('work_dir'), "cache")
 REPOSITORIES_DIR_NAME = 'repositories'
```

Comment:
Why keep the cache dir full path and repositories dir name? Be consistent, either use full paths in both or dir names in both.